openshift/4.0/templates/network-security-policy/nsp-tools.yaml

```diff
@@ -1,56 +1,51 @@
-    required: true
-    value: "354028"
-  - name: ENV_NAME
-    displayName: "Environment name"
-    description: "The name for this environment [dev, test, prod, tools]"
-    required: true
-    value: "tools"
+labels:
+  template: pims-tools-network-security-policy
+metadata:
+  name: pims-tools-network-security-policy
 objects:
-  # Allow project namespace to communicate with k8s cluster api.
-  - kind: NetworkSecurityPolicy
-    apiVersion: security.devops.gov.bc.ca/v1alpha1
+  - kind: NetworkPolicy
+    apiVersion: networking.k8s.io/v1
     metadata:
-      name: project-to-k8s-api-${ENV_NAME}
+      name: deny-by-default
     spec:
-      description: Allow pods to talk to the internal K8S api
-      source:
-        - - $namespace=${PROJECT_NAMESPACE}-${ENV_NAME}
-      destination:
-        - - int:network=internal-cluster-api-endpoint
-
-  # Allow tools project to communicate with dev, test, and prod.
-  - kind: NetworkSecurityPolicy
-    apiVersion: security.devops.gov.bc.ca/v1alpha1
+      # The default posture for a security first namespace is to
+      # deny all traffic. If not added this rule will be added
+      # by Platform Services during environment cut-over.
+      podSelector: {}
+      ingress: []
+  - apiVersion: networking.k8s.io/v1
+    kind: NetworkPolicy
     metadata:
-      name: project-to-project-permit-${ENV_NAME}-all
+      name: allow-from-openshift-ingress
     spec:
-      description: Allow project to communicate to another project.
-      source:
-        - - $namespace=${PROJECT_NAMESPACE}-${ENV_NAME}
-      destination:
-        - - $namespace=${PROJECT_NAMESPACE}-dev
-        - - $namespace=${PROJECT_NAMESPACE}-test
-        - - $namespace=${PROJECT_NAMESPACE}-prod
-
-  # Allow service accounts to run builds/deployments.
-  - kind: NetworkSecurityPolicy
-    apiVersion: security.devops.gov.bc.ca/v1alpha1
+      # This policy allows any pod with a route & service combination
+      # to accept traffic from the OpenShift router pods. This is
+      # required for things outside of OpenShift (like the Internet)
+      # to reach your pods.
+      ingress:
+        - from:
+            - namespaceSelector:
+                matchLabels:
+                  network.openshift.io/policy-group: ingress
+      podSelector: {}
+      policyTypes:
+        - Ingress
+  - kind: NetworkPolicy
+    apiVersion: networking.k8s.io/v1
     metadata:
-      name: service-account-to-k8s-api-${ENV_NAME}
+      name: allow-same-namespace
     spec:
-      description: Allow service accounts to run builds/deployments.
-      source:
-        - - $namespace=${PROJECT_NAMESPACE}-${ENV_NAME}
-          - "@app:k8s:serviceaccountname=builder"
-        - - $namespace=${PROJECT_NAMESPACE}-${ENV_NAME}
-          - "@app:k8s:serviceaccountname=deployer"
-      destination:
-        - - int:network=internal-cluster-api-endpoint
-        - - ext:network=any
+      # Allow all pods within the current namespace to communicate
```

Comment:
in the case of the tools namespace this seemed acceptable to me, let me know if you don't agree.